### DISCUSSION TITLE
How to Run Wiki.js on Hosting Account

### DISCUSSION THREAD

**Discussion by ds-tx** [requarks/wiki#5400] [Help / Questions]
I don't understand how to use Wiki.js on a hosting account.

I downloaded wiki-main.zip, uploaded to subfolder on my hosting account and unzipped it.
But it doesn't contain an index file of any sort.

**Comment by NGPixel** [comment]
Please read the installation intructions:

https://docs.requarks.io/install

**Comment by ds-tx** [comment]
I've seen those instructions and I still have no idea what to do.
Do I have to contact my hosting provider and find out if my hosting server is running Windows or Linux?
I cannot run shell scripts or terminal commands on my hosting server.
